rel/files/mongooseim.cfg

```
@@ -641,12 +641,10 @@
     [
         {service_admin_extra, [{submods, [node, accounts, sessions, vcard, gdpr,
                                           roster, last, private, stanza, stats]}]},
-        {service_mongoose_user_stats, []}
+        {service_mongoose_system_stats, []}
```

Adding the service in `rel/files/mongooseim.cfg` file will end up enabling reporting on all nodes, not only these used for tests. We most probably are not ready to enable this feature for all users yet.

---

I suggest to change the line to the one below
```suggestion
        {{{service_mongoose_system_stats}}}
```
remove the ending `,` above, and then in files like `mim*.vars.config` add entry like:
```
{service_mongoose_system_stats, "{service_mongoose_system_stats, []}"}.
```

---

Also make sure the release built with `rebar3 as prod release` or `make rel` doesn't have the service enabled.